src/inmanta/server/services/resourceservice.py

```diff
@@ -243,7 +244,7 @@ async def get_resource(
                 action_name = log_action.name
 
             actions = await data.ResourceAction.get_log(
-                environment=env.id, resource_version_id=resource_id, action=action_name, limit=log_limit
+                environment=env.id, resource_version_id=resource_id, action=action_name, limit=log_limit, connection=connection
```

Comment:
The call to `data.Resource.get()` in this method, is also a call to the database.